src/app/lib/rotary.js

```diff
@@ -20,6 +26,13 @@ export const updateWorkspaceMode = (mode = WORKSPACE_MODE.DEFAULT) => {
         const prev101Value = controller.settings.settings.$101;
         store.replace('workspace.rotaryAxis.prev101Value', prev101Value);
         controller.command('gcode', `$101=${ROTARY_AXIS_101_VALUE}`);
+
+        if (firmwareType === 'Grbl') {
+            controller.command('gcode', `$103=${ROTARY_AXIS_103_VALUE}`);
+            controller.command('gcode', `$113=${ROTARY_AXIS_113_VALUE}`);
+            controller.command('gcode', `$123=${ROTARY_AXIS_123_VALUE}`);
```

Comment:
Seems unnecessary to invoke the command function three separate times, you can just pass it an array of values, ex. `controller.command('gcode', ['gcode1', gcode2' gcode3']`)